### PR TITLE
[rmq][consumer] queue and exchange verification

### PR DESCRIPTION
**Pull request by Lunastryke**

### CODE REVIEW

File: rmq/consumer/connector.go [hunk 1]
```diff
@@ -82,6 +82,35 @@ func New(connConfig ConnectionConfig, queueConfig QueueConfig, queueBindConfig Q
...
+		if err != nil {
+			logger.Error("ERR_RMQ-CONSUMER_FAIL-CONNECT", err)
+			time.Sleep(connConfig.ReconnectInterval)
+			// Wait before retrying
+			continue
+		}
+		logger.Out("RMQ-CONSUMER", "Connected to RabbitMQ")
+		return conn, nil
+	}
+	return nil, fmt.Errorf("failed to connect to RabbitMQ after %d attempts", connConfig.ReconnectMaxAttempt)
```
Comment (kelchy) [inline rmq/consumer/connector.go:101]: what is the default max attempt?

File: rmq/consumer/config.go [hunk 2]
```diff
@@ -60,6 +60,30 @@ func DefaultConfig(name string) Config {
 	}
 }
 
+// ExchangeConfig is the configuration for exchange creation.
```
Comment (kelchy) [inline rmq/consumer/config.go:70]: where do you define the default behaviour?

File: rmq/consumer/connector.go (hunk 1 above)
Comment (Lunastryke) [reply rmq/consumer/connector.go:101]: Default is set to 3. 
`
// DefaultConnectionConfig returns a default connection configuration.
func DefaultConnectionConfig(connURIs []string) ConnectionConfig {
	return ConnectionConfig{
		ConnURIs:            connURIs,
		ReconnectInterval:   5 * time.Second,
		ReconnectMaxAttempt: 3,
	}
}`

File: rmq/consumer/config.go (hunk 2 above)
Comment (Lunastryke) [reply rmq/consumer/config.go:70]: Default behaviour is defined through the use of the `DefaultExchangeConfig( )` helper function, which requires only 2 params and generates a ExchangeConfig( ) object. If users do not use the helper, they will have to specify each field in the ExchangeConfig manually.

File: rmq/consumer/connector.go (hunk 1 above)
Comment (kelchy) [reply rmq/consumer/connector.go:101]: should we make that more obvious like at the top of file or in env or constants?

**Pull request merged**